x-pack/plugins/inference/common/tasks/nl_to_esql/ast/index.ts

```diff
@@ -4,5 +4,4 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-
```

Comment:
Can I get my newline back, pretty please?

Reply:
Absolutely @pgayvallet, sorry but rebasing this PR is -to say the least- always very painful 😂